docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+
+- **Knowing what is in your environment.** This requires discovering your dependencies and any transitive dependencies to understand the risks of those dependencies such as vulnerabilities or licensing restrictions.
+- **Manage your dependencies.** When a new security vulnerability is discovered, you must determine whether you are impacted, and if so, update to the latest version and security patch available. This is especially important to review changes that introduce new dependencies or regularly auditing older dependencies.
+- **Monitor your supply chain.** This is by auditing the controls you have in place to manage your dependencies. This will help you enforce more restrictive conditions to be met for your dependencies.
+
+![The State of the Octoverse 2020 - Advisories](media/advisories.png)
+
+We will cover various tools & techniques that NuGet & GitHub provides that you can use today to address potential risks inside your project. 
+
+## Knowing what is in your environment
+
+### NuGet Dependency Graph
+
+You can view your NuGet dependencies in your project by looking directly at the respective project file.
+
+This is typically found in one of two places:
+
+-	[`packages.config`](https://docs.microsoft.com/nuget/reference/packages-config) – Located in the project root.
+-	[`<PackageReference>`](https://docs.microsoft.com/nuget/consume-packages/package-references-in-project-files) – Located in the project file. 
+
+Depending on what method you use to manage your NuGet dependencies, you can also use Visual Studio to view your dependencies directly in the [Solution Explorer](https://docs.microsoft.com/visualstudio/ide/solutions-and-projects-in-visual-studio?view=vs-2019#solution-explorer) or [NuGet Package Manager](https://docs.microsoft.com/nuget/consume-packages/install-use-packages-visual-studio).
+
+For CLI environments, you can use the [`dotnet list package`](https://docs.microsoft.com/dotnet/core/tools/dotnet-list-package) command to list out your project or solution’s dependencies. 
+
+For more information on managing NuGet dependencies, [see the following documentation](https://docs.microsoft.com/nuget/consume-packages/overview-and-workflow).
+
+### GitHub Dependency Graph 
+
+You can use GitHub’s dependency graph to see the packages your project depends on and the repositories that depend on it. This can help you see any vulnerabilities detected in its dependencies.
+
+For more information on GitHub repository dependencies, [see the following documentation](https://github.co/dependency-graph).
+
+### Dependency Versions
+
+To ensure a secure supply chain of dependencies, you will want to ensure that all your dependencies are regularly updated to the latest stable version. Your dependencies can include code you depend on, binaries you consume, tooling you use, and other components. This can include:
```

Comment:
Consider adding: "Newer versions of your dependencies will often include patches of security vulnerabilities. Updating your dependencies to latest stable versions regularly will ensure you get those pathces."

Reply:
Added